dependencies.props

```diff
@@ -24,7 +24,7 @@
 
   <!-- Full package version strings that are used in other parts of the build. -->
   <PropertyGroup>
-    <CoreClrPackageVersion>2.0.0</CoreClrPackageVersion>
+    <CoreClrPackageVersion>2.0.1</CoreClrPackageVersion>
```

Comment:
This isn't the version you want to bump. it is the PackageVersion property above.